server/config.go

```diff
@@ -10,21 +10,19 @@ import (
 )
 
 const (
-	// ConfigCommandTrigger trigger for the command
-	ConfigCommandTrigger = "config"
+	configCommandTrigger = "config"
 	hint                 = "[vcs/org-name/project-name]"
 	helpText             = "View the config. Pass in the project (vcs/org/projectname) to set the default config"
```

Comment:
To not collide with other commands in the `main` package, I suggest
```suggestion
	configCommandHint                 = "[vcs/org-name/project-name]"
	configCommandHelpText             = "View the config. Pass in the project (vcs/org/projectname) to set the default config"
```

Reply:
done